docs/core/index.md

```diff
@@ -8,7 +8,7 @@ ms.custom: "updateeachrelease"
 ---
 # .NET Core Guide
 
-[.NET Core](about.md) is an [open-source](https://github.com/dotnet/coreclr/blob/master/LICENSE.TXT) general-purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It's cross-platform, supporting Windows, macOS and Linux, and can be used in device, cloud, and IoT applications.
+[.NET Core](about.md) is an [open-source](https://github.com/dotnet/coreclr/blob/master/LICENSE.TXT), general-purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It's cross-platform; supports Windows, macOS, and Linux; and can be used to build device, cloud, and IoT applications.
```

Comment:
- What makes .NET Core cross-platform is that it supports Windows, macOS, and Linux,; so cross-;platform and the list of platforms shouldn't be separated. The original sentence, with ", supporting" is correct. Please revert the change.
- the semicolon after Linux should be a comma. Please revert that change as well.